grakn-test/src/test/java/ai/grakn/test/graql/analytics/AnalyticsTest.java

```diff
@@ -903,6 +903,9 @@ public void testComputingUsingDegreeResource() throws GraknValidationException {
 
         // compute count
         analytics = new Analytics(graph.getKeyspace(), Collections.singleton("degree"), new HashSet<>());
+
+        //Not sure why the above seems to be closing the graph now.
+        graph = Grakn.factory(Grakn.DEFAULT_URI, graph.getKeyspace()).getGraph();
```

Comment:
Okay so from my understanding reopening the graph for the purpose of asserting the result is okay.